source/change-streams/tests/README.rst

```diff
@@ -49,7 +49,7 @@ Each YAML file has the following keys:
     - ``database``: Watch changes on database ``database_name``
     - ``client``: Watch changes on entire clusters
   - ``topology``: An array of server topologies against which to run the test.
-    Valid topologies are ``single``, ``replicaset``, and ``sharded``.
```

Comment:
Don't forget to bump changelogs for any test formats you're updating here.

Reply:
The Change Streams spec tests [README](https://github.com/mongodb/specifications/tree/master/source/change-streams/tests) doesn't include a change log, so I added an entry to the main test changelog. For retryable reads and writes, I added entries to the test README changelogs.